examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+# Kubernetes Deployment Guide
+
+## Setup Environment
+
+1. Start a Kubernetes cluster with version 1.15 or higher
+
+```
+$ minikube start --kubernetes-version v1.15.4
+😄  minikube v1.4.0 on Darwin 10.14.6
+🔥  Creating virtualbox VM (CPUs=2, Memory=4096MB, Disk=20000MB) ...
+🐳  Preparing Kubernetes v1.15.4 on Docker 18.09.9 ...
+🚜  Pulling images ...
+🚀  Launching Kubernetes ...
+⌛  Waiting for: apiserver proxy etcd scheduler controller dns
+🏄  Done! kubectl is now configured to use "minikube"
+```
+
+2. Download the latest version of Kuma
```

Comment:
Per @subnetmarco feedback, he would like demo application to be installed and demoed before `Kuma` control plane is installed on Kubernetes cluster

Reply:
I moved the demo application installation ahead of kuma installation